src/raps/common.ts

```diff
@@ -228,18 +267,32 @@ const executeAction = async (
   baseNonce?: number
 ): Promise<RapActionResponse> => {
   logger.log('[1 INNER] index', index);
-  const { parameters, type } = action;
-  const actionPromise = findActionByType(type);
-  logger.log('[2 INNER] executing type', type);
+  const { type, parameters } = action;
+  let nonce;
   try {
-    const nonce = await actionPromise(
-      wallet,
-      rap,
-      index,
-      parameters,
-      baseNonce
-    );
-    return { baseNonce: nonce, errorMessage: null };
+    logger.log('[2 INNER] executing type', type);
+    const rapType = getRapTypeFromActionType(type);
+    if (rapType === RAP_TYPE.ENS) {
```

Comment:
I don't really like this pattern to choosee action raps so I may change it in another PR